src/annotations/standard/ParamAnnotation.php

```diff
@@ -53,7 +53,10 @@ public static function parseAnnotation($value)
     {
         $parts = explode(' ', trim($value), 3);
 
-        return array('type' => $parts[0], 'name' => substr($parts[1], 1));
+        if (isset($parts[1]))
+            return array('type' => $parts[0], 'name' => substr($parts[1], 1));
+        else
+            return array('type' => 'unknown', 'name' => substr($parts[0], 1));
```

Comment:
I think we should use empty string instead of `unknown`, because user haven't typed anything as data type and anything != `unknown` word.